.github/actions/ci-setup/action.yml

```diff
@@ -1,11 +1,12 @@
 name: 'CI setup'
+description: 'Installs Node.js and package dependencies.'
 runs:
   using: 'composite'
   steps:
-    - name: Setup Node.js 16.x
-      uses: actions/setup-node@v3
+    - name: Setup Node.js 20.x
```

Comment:
yesterday, Node LTS Version is updated to 22, is it ok?
https://nodejs.org/en/about/previous-releases